lib/v2/hunau/utils/Common.js

```diff
@@ -0,0 +1,61 @@
+// common.js
+const cheerio = require('cheerio');
+const got = require('@/utils/got');
+const Category = require('./Category');
+const NewsContent = require('./NewsContent');
+const IndexPage = require('./IndexPage');
+
+async function GetContent(ctx, { baseHost, baseCategory, baseType, baseTitle, baseDescription = '', baseDeparment = '', baseClass = 'div.article_list ul li:has(a)' }) {
+    const { category = baseCategory, type = baseType, page = '1' } = ctx.params;
+
+    const title = `${baseTitle} - ${Category(category)}`;
+    const description = baseDescription ? `${baseDescription} - ${Category(category)}` : title;
+
+    const typeURL = type ? `/${type}` : '';
+    const baseURl = `${baseHost}${typeURL}/${category}`;
+    const url = `${baseURl}/${IndexPage(page)}`;
+
+    const { data: response } = await got(url);
+    const $ = cheerio.load(response);
+
+    const list = $(baseClass)
+        .toArray()
+        .map((item) => {
+            item = $(item);
+
+            const a = item.find('a');
+            const href = a.attr('href');
+            const title = a.text();
+            const link = href.startsWith('./') && !href.endsWith('.pdf') ? `${baseURl}${href.replace('./', '/')}` : href;
+
+            return {
+                title,
+                link,
+            };
+        });
+
+    const items = await Promise.all(
+        list.map((item) =>
+            ctx.cache.tryGet(item.link, async () => {
+                const newsContent = await NewsContent(item.link, baseDeparment);
+
+                item.pubDate = newsContent.pubDate;
+                item.description = newsContent.description;
+
+                return item;
+            })
+        )
+    );
+
+    ctx.state.data = {
+        // 源标题
+        title,
+        description,
+        // 源链接
+        link: String(url),
```

Comment:
https://github.com/DIYgod/RSSHub/blob/9bdf0364f36b3afaf1004488958c8ec420c82d8a/lib/v2/hunau/utils/Common.js#L16
`url` is already a `string`. Don't type cast the same thing again.
docs: https://docs.rsshub.app/joinus/script-standard.html